example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
+        #
+        # If you do not want this bonus to be applied for neurons with higher
+        # dissolve delay, set `bonus` to `0%` and those neurons will not receive
+        # higher voting power.
         DissolveDelay:
+            # This parameter sets the maximum dissolve delay a neuron can have.
+            # When reached, the maximum dissolve delay bonus will be applied.
             duration: 8 years
+            # If a neuron's dissolve delay is `duration`, its voting power will
+            # be multiplied by this `bonus` amount.
             bonus: 100%
 
+        # Users with neurons staked in the non-dissolving state for a long
+        # period of time are incentivized to take the long-term interests of
+        # the SNS into consideration when voting. To reward this long-term
+        # commitment, this bonus can be set to a percentage (greater than zero),
+        # which will result in neurons having their voting power increased in
+        # proportion to their age.
+        #
+        # If this bonus should not be applied for older neurons, set the bonus
+        # to `0%` and older neurons will not receive higher voting power.
         Age:
+            # This parameter sets the duration of time the neuron must be staked
+            # in the non-dissolving state, in other words its `age`, to reach
+            # the maximum age bonus. Once this age is reached, the neuron will
+            # continue to age, but no more bonus will be applied.
             duration: 4 years
+            # If a neuron's age is `duration` or older, its voting power will be
+            # multiplied by this `bonus` amount.
             bonus: 25%
 
+    # TODO[ATG]: The current description doesn't explain how much rewards will
+    #
+    # there be for a particular action given a particular `RewardRate`
+    # configuration. How exactly does inflation project to voting rewards?
+    #
+    # The voting reward rate controls how quickly the supply of the SNS token
+    # increases. For example, setting `initial` to `2%` will cause the supply to
+    # increase by at most `2%` per year. A higher voting reward rate
+    # incentivizes users to participate in governance, but also results in
+    # higher inflation.
+    #
+    # The initial and final reward rates can be set to have a higher reward rate
+    # at the launch of the SNS and a lower rate further into the SNS’s lifetime.
+    # The reward rate falls quadratically from the `initial` rate to the `final`
+    # rate over the course of `transition_duration`.
+    #
+    # Setting both `initial` and `final` to `0%` will result in the system not
+    # distributing voting rewards at all.
     RewardRate:
+        # The initial reward rate that SNS voting rewards will increase the SNS
+        # token supply per year.
         initial: 10%
+
+        # The final reward rate that SNS voting rewards will increase the SNS
```

Comment:
should it be "rate at which the SNS rewards will increase"? 
(not sure I understand this)

Reply:
Is this okay?

```
        # The initial reward rate at which the SNS voting rewards will increase
        # per year.
        initial: 10%

        # The final reward rate at which the SNS voting rewards will increase
        # per year. This rate is reached after `transition_duration` and remains
        # at this level unless changed by an SNS proposal.
        final: 2.25%
```